inspector/src/components/actionTabs/tabs/propertyGrids/animations/timeline.tsx

```diff
@@ -71,11 +71,15 @@ export class Timeline extends React.Component<
 
   calculateScrollWidth(start: number, end: number) {
     if (this._scrollContainer.current && this.props.animationLimit !== 0) {
-      const containerWidth = this._scrollContainer.current.clientWidth;
+      const containerWidth = this._scrollContainer.current.clientWidth - 6;

```

Comment:
Why -6 pixels?

Reply:
ah, that is the padding of the scrollbar. Since I'm calculating the parent width, I don't exactly want the whole size but want to give some extra padding. Need to add a comment there...

Reply:
Is this part of the design?

Reply:
Yes. Basically represents this padding between the scrollbar and its parent.. 
![image](https://user-images.githubusercontent.com/704682/86146771-41e06900-babe-11ea-8804-67bb5eec0459.png)